data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,444 @@ mission "Heliarchs Buy Jump Drive Mission"
+		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg. Also found out she was ambidextrous."
+		"salary: Anna Fangvin" = 5000
+		payment 2000000
+		set "beyond the hyperdrive news can show now"
+	`The moment you land, Anna is all too happy to see you. She grabs your shoulders with both arms and in an excited tone explains that her book has been approved for publication.`
+	`	"Today, they will be publishing 'Beyond the Hyperdrive' and I will be signing copies! Thanks for everything you've done, <first>!"`
+	choice
+		`	"It got approved? That's awesome!"`
+	`	"I know, right? So do you want to join me during the book signing?"`
+	choice
+		`	"I've been with you through the whole process. Count me in!"`
+	`	"Great! Let's walk to the bookstore and prepare everything! I already have my pen ready, and the staff have already prepared seats for the both of us."`
+	`	You and Anna walk into the massive bookstore at the spaceport. There are already several Heliarchs present, their Enforcer Riot Staffs in place for crowd control.`
+	`	A Kimek, dressed in what looks like the bookstore's uniform, takes out the Kimek equivalent of a loudspeaker and begins announcing the release of 'Beyond the Hyperdrive'. Within ten minutes, the bookstore has become crowded, with a long queue forming between the cashiers and your table, as Anna signs every copy brought to her by each individual Kimek.`
+	`	Occasionally, Anna alternates between her left and right hand to sign books whenever the other is tired. For hours, this continues. You tried to keep count of how many copies she signed, but gave up when it reached six hundred.`
+	choice
+		`	"Are you left or right handed?"`
+	`	"I was originally born left-handed, but as a child I chose to learn how to use my right hand, because I thought being able to use both hands would be cool."`
+	`	The signing continues for what feels like four hours. Then, the same Kimek in the bookstore uniform announces that the store is to close within an hour.`
+	`	As the store closes and the queue shortens, Anna signs the last Kimek's copy. With that, the event is over. Anna then tells you to meet her at your ship. You and Anna walk back to your ship. You open the hatch to allow Anna inside, and you both walk through.`
+	`	"Today was a resounding success. My book succeeded beyond my wildest dreams, and it was all because of you <first>!", she says, almost in a teary voice. She pulls out a credit chip. "Here, two million credits. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 5,000 credits daily, a small but significant portion of my royalties."`
```

Comment:
```suggestion
	`	"Today was a resounding success. My book succeeded beyond my wildest dreams, and it was all because of you <first>!" she says, almost in a teary voice. She pulls out a credit chip. "Here, <payment>. You've done so much for me and my career, and you deserve it. Additionally, you will be receiving a salary of 5,000 credits daily, a small but significant portion of my royalties."`
```